nvflare/fed_job.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
This file should not be located at "nvflare/", maybe moved to "nvflare/job_config" folder?

Reply:
I think we should have a folder for this one,
either the same inside "nvflare/client"
or we have something as Yuhong suggest "nvflare/job_config" or "nvflare/job"

Reply:
Agreed. Moving it to nvflare/job_config